docs/knowledge_base/tips/colab_with_ngrok/index.md

```diff
@@ -159,7 +167,7 @@ Here are the new cells to add:
    gui=Gui(pages=pages)
    ```
 
-2. Update your `tp.run(gui)`:  
+2. Update your `tp.run(gui)`:
```

Comment:
?

Reply:
@FlorianJacta I believe this comment is for you.

Reply:
I don't know what it means. I believe this article is from @marisogo 

But we could put:

2. Run your application: